Geodatabase Data Curation Primer/Geodata-Primer.md

```diff
@@ -297,7 +297,10 @@ In the instance of geodatabases, curators may wish to create an individual recor
 Geodatabases are a proprietary format created by ESRI. While QGIS can open them, most other GIS analysis tools cannot. It should be noted that while ESRI is the industry leader in GIS software and in heavy use, other ESRI formats have already become obsolete. It is therefore recommended, where possible, to export files within a geodatabase to formats documented as suitable for long-term preservation.  In these cases, it is recommended that both the geodatabase and the conversion files are kept and documented in an accompanying README file.  Below are transformation recommendations and guidelines for the major types of files within a geodatabase.
 
   **Shapefiles**<br/>
-  Shapefiles are the most persistent geospatial vector format thus far, so transforming a feature layer from a geodatabase to a shapefile may be a good idea. However, though they are ubiquitous and many softwares can open them, shapefiles are still technically proprietary. Note: shapefiles are not able to hold as many columns as a geodatabase feature layer, so a single geodatabase may need to be converted into multiple shapefiles. Another option to explore is exporting the feature layer to a CSV, which is very doable for point data, but note that for polygons and lines information could be lost. A good idea is to keep the geodatabase and do the shapefile and csv conversions.
```

Comment:
I removed the sentence about exporting feature layers to CSV.  I think its useful to include as a preservation option but not sure it fits in the shapefiles section.  There are some additional instructions in a section below that describes it well.

Reply:
Do you think this would be a good place to mention the file size limitations of shapefiles, and the fact that the ability of the curator to convert GDB feature classes (where file sizes can be much bigger) into separate shapefiles might be constrained by these limits? I think the "where possible" is a little cryptic, and it may be worthwhile to be explicit about the main reason why exporting may not be possible.

Reply:
hmmm...looks like when I agreed to merge your pull request it just wrote over all of the changes that I had made?  The text shown now is just the original, not the changes I tried to commit..

Reply:
I agree that it would be good to edit the geodatabase paragraph discuss in what cases transformation would be recommended -  warning curators to think about limitations of long-term preservation file formats and how to describe/preserve internal relationships between data layers in more complicated geodatabases.

Reply:
I committed a small change, and I think these directives are clear enough throughout the primer